spinalcordtoolbox/scripts/sct_compute_ernst_angle.py

```diff
@@ -164,8 +164,8 @@ def main(argv: Sequence[str]):
             f = open(fname_output_file, 'w')
             f.write(str(graph.getErnstAngle(input_tr)))
             f.close()
-        except:
-            printv('\nERROR: Cannot open file'+fname_output_file, '1', 'error')
+        except OSError:
+            printv(f"\nERROR: Cannot open file {fname_output_file}", '1', 'error')
```

Comment:
Re: 3f0881d1c2905deb2df669b969338f6a7f1d81f3, is it possible we just want to remove the try/except entirely, and let the code throw its own `OSError`?

I guess the current code does remove the traceback and make the output a little cleaner/user-friendly? 

At the same time, if this script is somehow failing to write to its output file, it would be a rare enough circumstance that we'd _want_ to see a detailed traceback for debugging purposes.

Reply:
I was going to protest that the traceback would be missing the filename, but I just testes and actually it's there, and it also specifies whether it's a `IsADirectoryError` or a `PermissionError` or a `FileNotFoundError`, which is helpful. Fixed.